[2022-08-23 11:37:16,331] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-09T12:00:00+00:00 [queued]>
[2022-08-23 11:37:16,354] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-09T12:00:00+00:00 [queued]>
[2022-08-23 11:37:16,355] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:37:16,356] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 11:37:16,357] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:37:16,389] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): print_world> on 2021-10-09 12:00:00+00:00
[2022-08-23 11:37:16,399] {standard_task_runner.py:52} INFO - Started process 334 to run task
[2022-08-23 11:37:16,404] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'print_world', 'scheduled__2021-10-09T12:00:00+00:00', '--job-id', '1570', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmp7whmnzx_', '--error-file', '/tmp/tmp2_rpoy0c']
[2022-08-23 11:37:16,405] {standard_task_runner.py:80} INFO - Job 1570: Subtask print_world
[2022-08-23 11:37:16,510] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-09T12:00:00+00:00 [running]> on host 7f48b7aaf0fc
[2022-08-23 11:37:16,618] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=print_world
AIRFLOW_CTX_EXECUTION_DATE=2021-10-09T12:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-09T12:00:00+00:00
[2022-08-23 11:37:16,619] {logging_mixin.py:115} INFO - world
[2022-08-23 11:37:16,620] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 11:37:16,641] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=print_world, execution_date=20211009T120000, start_date=20220823T113716, end_date=20220823T113716
[2022-08-23 11:37:16,698] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 11:37:16,789] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
